wordpress-mobile/gutenberg-mobile
# Suppress the List block on Android

We've found some issues with the list block support on Android (the https://github.com/wordpress-mobile/gutenberg-mobile/issues/898 is the blocker) so, we're suppressing the list block for Android for now, until we fix those.

Gutenberg side PR: https://github.com/WordPress/gutenberg/pull/15100

To test:

1. Run the demo app on Android
2. Notice that the list block in the end of the example content is unsupported (only it's block Name/icon is shown)
3. Notice that there is no "List block" in the Inserter
4. Run the demo app on iOS and notice that the list block is supported there and available in the Inserter
